.changeset/eight-bats-double.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
The workspace resolver package likely needs a changeset for this

Reply:
good catch